src/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# @file build.sh
+# @author Jan Rodak - xrodak00
+# @brief script pro vytvoreni deb balicku
```

Comment:
U doxygen komentářů dodržujeme konvenci 
""" 
@file bla bla
@author bla bla
atd...
"""
Přepiš to pls